From 0b1a79c94aad799440f317d8d2385a863eefec5a Mon Sep 17 00:00:00 2001
From: Vinu Vaghasia <vinu.vaghasia@amd.com>
Date: Mon, 10 Jan 2022 16:40:17 -0600
Subject: [PATCH 1/1] ARM:dts:aspeed: Modify Onyx dts for iio-channel issue

io-channel-cell property added to eanble iio-channel for adc sensors

Signed-off-by: Vinu Vaghasia <vinu.vaghasia@amd.com>
---
 arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts b/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
index 51a4ae955dbb..f300522ba7ff 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
@@ -496,31 +496,37 @@
 			adc0: fpga_vdd_core@50 {
 				compatible = "ti,adc121c";
 				reg = <0x50>;
+				#io-channel-cells = <1>;
 				label = "fpga_vdd_core";
 			};
 			adc1: fpga_vdd_18@51 {
 				compatible = "ti,adc121c";
 				reg = <0x51>;
+				#io-channel-cells = <1>;
 				label = "fpga_vdd_18";
 			};
 			adc2: fpga_vdd_33@52 {
 				compatible = "ti,adc121c";
 				reg = <0x52>;
+				#io-channel-cells = <1>;
 				label = "fpga_vdd_33";
 			};
 			adc3: lom_vdd_33@53 {
 				compatible = "ti,adc121c";
 				reg = <0x53>;
+				#io-channel-cells = <1>;
 				label = "lom_vdd_33";
 			};
 			adc4: lom_vdd_12@54 {
 				compatible = "ti,adc121c";
 				reg = <0x54>;
+				#io-channel-cells = <1>;
 				label = "lom_vdd_12";
 			};
 			adc5: vdd_12_dual@55 {
 				compatible = "ti,adc121c";
 				reg = <0x55>;
+				#io-channel-cells = <1>;
 				label = "vdd_12_dual";
 			};
 		};
--
2.17.1

